engines/rancher_on_eks/app/models/rancher_on_eks/deployment.rb

```diff
@@ -155,6 +155,7 @@ def deploy
 
       step(1) do
         @vpc = AWS::Vpc.create()
+        @vpc.modify_vpc_attrs
         @vpc.wait_until(:available)
```

Comment:
Wouldn't the wait be before the modify?

Reply:
No, it did not matter to modify the VPC while creating it but it is better after creation, if an exception is raised (like VPC limit raise, or the command threw stderr), similar to the [gateway](https://github.com/SUSE-Enceladus/suse-rancher-setup/blob/main/engines/rancher_on_eks/app/models/rancher_on_eks/deployment.rb#L177) it would not reach the modify command. The modification can be done while the resource is being created (both modify-while-creating and modify-after-creation have been tested fine)